src/unity/toolkits/CMakeLists.txt

```diff
@@ -21,5 +21,6 @@ subdirs(ml_model
         image
         neural_net
         object_detection
+        one_shot_object_detection
```

Comment:
Hmm, any reason `one_shot_object_detector` can't just go into the `object_detection` directory?

Reply:
Done.